README.md

```diff
@@ -68,6 +68,7 @@ Options:
   -i <amount>       increase volume
   -l                use fullcolor instead of symbolic icons
   -m                toggle mute
+  -M <mixer>        specify mixer (ex: Headphone), default Master
```

Comment:
I'd only modify this to note that the option is specific to amixer, and put it more similar style as the other options. Here's a suggestion:
```suggestion
  -M <mixer>        specify mixer (amixer only, default: Master)
```